git_machete/tests/functional/test_machete.py

```diff
@@ -2041,7 +2042,7 @@ def test_github_create_pr(self) -> None:
         expected_error_message = "Pull request for branch hotfix/add-trigger is already created under link www.github.com!\nPR details: PR #6 by github_user: hotfix/add-trigger -> master"
         machete_client.read_definition_file()
         with self.assertRaises(MacheteException) as e:
-            machete_client.create_github_pr(head='hotfix/add-trigger', opt_draft=False, opt_onto=None)
+            machete_client.create_github_pr(head=LocalBranch.of('hotfix/add-trigger'), opt_draft=False, opt_onto=None)
```

Comment:
```suggestion
            machete_client.create_github_pr(head=LocalBranch('hotfix/add-trigger'), opt_draft=False, opt_onto=None)
```
In this case we don't have to worry about `None` option